.NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimexUtility.cs

```diff
@@ -457,6 +484,13 @@ public static string ModifyAmbiguousCenturyTimex(string timex)
             return "XX" + timex.Substring(2);
         }
 
+        public static float ParseNumberFromDurationTimex(string timex)
+        {
+            char[] unitChar = new char[] { 'D', 'W', 'M', 'Y', 'B' };
+            var numberStr = timex.Substring(timex.IndexOf('P') + 1, timex.IndexOfAny(unitChar) - 1);
```

Comment:
'P' should reference an existing constant instead of hardcoded here.

Reply:
done in #2930